onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -220,12 +222,26 @@ Status MatMulNBits::PrePack(const Tensor& tensor, int input_idx, /*out*/ Allocat
     }
     auto qptr = tensor.DataRaw();
     packed_b_ = IAllocator::MakeUniquePtr<void>(alloc, packed_b_size_, true);
-    MlasSQNBitGemmPackQuantBData(N_, K_, nbits_, block_size_, compute_type, qptr, packed_b_.get());
+    MlasSQNBitGemmPackQuantBData(N_, K_, nbits_, block_size_, compute_type, qptr, packed_b_.get(), nullptr, has_zp_input_, nullptr, nullptr);
     if (prepacked_weights) {
+      // TODO: cannot use packed_b_ after following code with std::move.
+      assert(false);
```

Comment:
why assert false here?

Reply:
this code shall not be reached. this is legacy code. I may remove it.